src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -227,50 +224,12 @@ const MessageContentWithContext = <
 
   if (message.deleted_at) {
     return (
-      <View
+      <MessageDeleted
+        getDateText={getDateText}
```

Comment:
Can we instead just pass `formattedData`? To keep it consistent with MessageHeader and MessageFooter